dlib/test/serialize.cpp

```diff
@@ -1119,6 +1121,9 @@ namespace
         t1.o.insert("hello from unordered_multiset");
         t1.o.insert("hello from unordered_multiset");
         t1.ptr_shared1 = make_shared<string>("hello from shared_ptr");
+	dlib::rand rng(std::time(NULL));
```

Comment:
Please remove all the other tabs too though :)

Reply:
Sorry. Hopefully this is it. 24 hour turnaround is long between reviews. Sodding time zones... It's almost as long as waiting for some VHDL to build :)